pipeline/metadata/schema.py

```diff
@@ -250,35 +250,35 @@ def flatten_for_bigquery_satellite(row: SatelliteRow) -> Dict[str, Any]:
   """Convert a structured satellite dataclass into a flat dict."""
   flat: Dict[str, Any] = {
       'domain': row.domain,
-      'category': row.category,
-      'ip': row.ip,
+      'domain_category': row.category,
+      'is_control_domain': row.is_control,
       'date': row.date,
       'start_time': row.start_time,
       'end_time': row.end_time,
-      'error': row.error,
-      'anomaly': row.anomaly,
+      'resolver_ip': row.ip,
+      'resolver_name': row.ip_metadata.name,
+      'is_control_resolver': row.is_control_ip,
```

Comment:
I'd use `resolver_is_trusted`.
`control` isn't clear, and better to have the `resolver_` prefix on all resolver fields.

Reply:
What about this?

Reply:
switched to resolver_is_trusted